File: c#/program17.cs
using System;

class program17
{
    public static int Factorial(int No)
    {
       int iFact = 1;

       for(int iCnt = 1; iCnt <= No; iCnt++)
       {
            iFact = iFact * iCnt;
       }
       return iFact;
    }
    
    public static void Main(string[] args)
    {
        int value = 0, ret = 0;

        Console.WriteLine("Enter number :");
        value = int.Parse(Console.ReadLine());

        ret = Factorial(value);
        Console.WriteLine("Result is = {0}",ret);
    }
}

File: java/program17.java
import java.util.*;

class Number
{
    public int iNo;

    public Number(int i)
    {
        iNo = i;
    }

    public int Factorial()
    {
        int iFact = 1;

        for(int icnt = 1; icnt <= iNo; icnt++)
        {
            iFact = iFact * icnt;
        }
        return iFact;
    }
}

class program17
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int iValue = sobj.nextInt();

        Number obj = new Number(iValue);
        System.out.println("Factorial is : "+obj.Factorial());
    }
}